include/illixr/vk_util/render_pass.hpp

```diff
@@ -40,7 +42,7 @@ class timewarp : public render_pass {
      * @param input_texture_vulkan_coordinates Whether the input texture is in Vulkan coordinates.
      */
     virtual void setup(VkRenderPass render_pass, uint32_t subpass, std::array<std::vector<VkImageView>, 2> buffer_pool,
-                       bool input_texture_vulkan_coordinates = true) = 0;
```

Comment:
Why is this removed? This would flip the texture.

Reply:
It was removed because default arguments on virtual functions are strongly discouraged due to potential issues with overloaded versions of the functions. All this change should do is force all of the calls that relied on the default value to add true to their argument list.

Reply:
Would you also be able to make the corresponding changes in monado_vulkan_integration since this function is used there? Otherwise, we could remove the virtual qualifier and add back the default.

Reply:
Also, `native_renderer` uses this function as well. So let's just remove the virtual and add back the default value.